### PR TITLE
DOC: Add examples of read_rect and read_bounds in wsi_reader notebook

### PR DESCRIPTION
**Pull request by Srijay-lab**
Added examples for read_rect and read_bounds. There's bug we spotted in read_region method so didn't add it. The method is failing on every "level" argument.

### CODE REVIEW

File: examples/04_example_patchextraction.ipynb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (shaneahmed) [inline examples/04_example_patchextraction.ipynb:1]: <p>You are not running the latest version of tiatoolbox to run this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

**Review by DavidBAEpstein — COMMENTED** [review]
[david_review_147.txt](https://github.com/TIA-Lab/tiatoolbox/files/7229869/david_review_147.txt)

File: examples/04_example_patchextraction.ipynb (hunk 1 above)
Comment (Srijay-lab) [reply examples/04_example_patchextraction.ipynb:1]: <p>Checked with latest version 0.7.0</p>

Comment (shaneahmed) [inline examples/04_example_patchextraction.ipynb:1]: <p>Please run the notebook with tiatoolbox 0.7.0. One way is to run on Google Colab.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/147/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (Srijay-lab) [reply examples/04_example_patchextraction.ipynb]: <p>Tested </p>

**Review by shaneahmed — APPROVED** [review]
Thanks @Srijay-lab 
Please let me know once @mostafajahanifar has approved it. I will merge it then.

**Review by mostafajahanifar — APPROVED** [review]
Everything looks good. Thanks @Srijay-lab.
I just edited example 1 based on David's comments.

**Comment by Srijay-lab** [comment]
> Everything looks good. Thanks @Srijay-lab. I just edited example 1 based on David's comments.

Thank you very much Mostafa. @shaneahmed I think we are ready to merge this branch.

**Pull request merged**